pkg/console/starter/starter.go

```diff
@@ -145,11 +173,13 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		"console",
 		[]configv1.ObjectReference{
 			{Group: "operator.openshift.io", Resource: "consoles", Name: api.ConfigResourceName},
-			{Group: "config.openshift.io", Resource: "consoles", Name: api.ConfigResourceName},
-			{Group: "config.openshift.io", Resource: "infrastructures", Name: api.ConfigResourceName},
+			{Group: configv1.GroupName, Resource: "consoles", Name: api.ConfigResourceName},
+			{Group: configv1.GroupName, Resource: "infrastructures", Name: api.ConfigResourceName},
 			{Group: "oauth.openshift.io", Resource: "oauthclients", Name: api.OAuthClientName},
+			{Group: configv1.GroupName, Resource: "oauths", Name: api.ConfigResourceName},
 			{Resource: "namespaces", Name: api.OpenShiftConsoleOperatorNamespace},
 			{Resource: "namespaces", Name: api.OpenShiftConsoleNamespace},
+			{Resource: "namespaces", Name: api.OpenshiftConfigManagedNamespace},
```

Comment:
I believe since we are using the `Syncer` to copy the configmap into our namespace, we don't have to list the `openshift-config-managed` namespace here.  The implication is that we care about ALL the things in that namespace.  Since we copy over the configmap, we don't really care about anything else in the namespace.